templates/ts/notification-restify/teamsapp.testtool.yml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.3/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.3
+
+deploy:
+  # Install development tool(s)
+  - uses: devTool/install
+    with:
+      testTool:
+        version: ~0.1.0
```

Comment:
our next release version for test tool should be `~0.1.0-beta`?

Reply:
Tested `~0.1.0` won't work for 0.1.0-beta. Let me fix this